test/conservation_checker_tests.jl

```diff
@@ -26,173 +25,177 @@ struct TestAtmos{I} <: Interfacer.AtmosModelSimulation
     i::I
 end
 name(s::TestAtmos) = "TestAtmos"
-get_field(s::TestAtmos, ::Val{:F_radiative_TOA}) = ones(s.i.space) .* 200
```

Comment:
make these multiline functions that get FT from `s.i.space`, instead of using FT from global scope